lnwire/update_fulfill_htlc.go

```diff
@@ -75,8 +90,7 @@ func (c *UpdateFulfillHTLC) MsgType() MessageType {
 //
 // This is part of the lnwire.Message interface.
 func (c *UpdateFulfillHTLC) MaxPayloadLength(uint32) uint32 {
-	// 32 + 8 + 32
-	return 72
+	return 65533
```

Comment:
use max msg size constant? or maybe we just remove it entirely since they all now return the same constant

Reply:
Would be in favor of removing it!

Reply:
Made it into a constant, IMO still useful if you're reeding out messages from a reader other than the underlying brontide transport.